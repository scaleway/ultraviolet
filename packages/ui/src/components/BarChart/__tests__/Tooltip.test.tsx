import { shouldMatchEmotionSnapshot } from '@utils/test'
import { describe, test } from 'vitest'
import BarChartTooltip from '../Tooltip'

describe('barChartTooltip', () => {
  test('renders correctly', () =>
    shouldMatchEmotionSnapshot(
      <BarChartTooltip
        color="#ff0000"
        formattedValue="10 kb"
        indexValue="2020-02-20"
      />,
    ))
})
