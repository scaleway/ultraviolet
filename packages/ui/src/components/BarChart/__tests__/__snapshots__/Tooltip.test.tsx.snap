// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`barChartTooltip > renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background: #ffffff;
  border-radius: 0.125rem;
  box-shadow: 0px 4px 32px 8px #d9dadd66;
  padding: 0.5rem 1rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.emotion-2 {
  display: block;
  width: 0.75rem;
  height: 0.75rem;
  background: #ff0000;
  margin-right: 0.75rem;
}

<div
    data-testid="testing"
  >
    <div
      class="emotion-0 emotion-1"
    >
      <div>
        <span
          class="emotion-2 emotion-3"
        />
      </div>
      <div>
        <p
          class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_sentiment_primary__m4c9ow9 style_prominence_default__m4c9owi style_variant_bodyStronger__m4c9owr style_disabled_false__m4c9ow1b style_undefined_compound_0__m4c9ow1c style_undefined_compound_72__m4c9ow3c"
        >
          10 kb
        </p>
        <p
          class="style__m4c9ow0 style_strikeThrough_false__m4c9ow2 style_italic_false__m4c9ow4 style_underline_false__m4c9ow6 style_oneLine_false__m4c9ow8 style_prominence_default__m4c9owi style_variant_bodySmall__m4c9own style_disabled_false__m4c9ow1b style_undefined_compound_72__m4c9ow3c"
        >
          2020-02-20
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
